### PR TITLE
Debug/andrew/agent interface

### PR DESCRIPTION
**Pull request by ajn2004**
Fixes some issues that kept the FLAT agent from running smoothly.

### CODE REVIEW

File: src/llm_vm/agents/FLAT/agent_helper/bothandler.py [hunk 1]
```diff
@@ -2,17 +2,17 @@
 import openai
 import random
 from llm_vm.utils.typings_llm import *
```
Comment (cartazio) [inline src/llm_vm/agents/FLAT/agent_helper/bothandler.py:4]: i dont know how i missed this in my patch, good catch

File: src/llm_vm/completion/data_synthesis.py [hunk 2]
```diff
@@ -1,20 +1,86 @@
 import json
 import sys 
+from sentence_transformers import SentenceTransformer, util
+
 
 class DataSynthesis:
```
Comment (cartazio) [inline src/llm_vm/completion/data_synthesis.py:6]: oh is your branch ontop of Viks? i thought that was merged right?

File: src/llm_vm/completion/test_data_synthesis.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from dotenv import load_dotenv
```
Comment (cartazio) [inline src/llm_vm/completion/test_data_synthesis.py:1]: i thought we already had this file... with an entry point registration

**Review by cartazio — COMMENTED** [review]
i'm confused, it looks like some merge you did clobbered history ?

File: src/llm_vm/completion/data_synthesis.py (hunk 2 above)
Comment (ajn2004) [reply src/llm_vm/completion/data_synthesis.py:6]: it says I'm consistent with the main branch right now I think

**Pull request merged**